conda/tvm/meta.yaml

```diff
@@ -41,6 +41,7 @@ requirements:
     - python
     - {{ pin_compatible('numpy') }}
     - decorator
+    - astor
```

Comment:
Could someone more experienced with managing TVM's dependencies tell me if this is the right manner to indicate a new Python dependency? `astor` is only needed for testing (I could possibly exclude it entirely, though it would be convenient to have it)

Reply:
Nevermind, @tqchen suggested it's better to exclude the new dependency if it's not used directly in tests (it shouldn't be; `astor` was only meant for debugging)